### PR TITLE
various frontend tweaks to icons and wording

### PR DESCRIPTION
**Pull request by holtzman**
* updated icons: wrap + mint, burn, burn + unwrap, wrap, pending, and error
* changed icon size for eUSD and RSV when denominating amounts to be consistent and match size when used elsewhere in table row
* changed wording in some places: safe -> custodian multisig, and in type column for unaudited rows

### CODE REVIEW

File: frontend/src/components/CopyableField.tsx [hunk 1]
```diff
@@ -10,17 +10,24 @@ export default function CopyableField({
...
 }) {
+
+  if (!copyText) {
+    copyText = text
+  }
+
   function copyToClipboard(e: React.MouseEvent<HTMLButtonElement>) {
     e.stopPropagation()
-    navigator.clipboard.writeText(text)
+    navigator.clipboard.writeText(copyText)
```
Comment (Shramp) [inline frontend/src/components/CopyableField.tsx:30]: You can avoid the `if` statement and the variable reassignment here by doing:
```
navigator.clipboard.writeText(copyText ?? text)
```

`??` evaluates to the second variable if the first one is null or undefined

File: frontend/src/components/WrapTableRow.tsx [hunk 2]
```diff
@@ -47,6 +49,24 @@ const SpacerRow = () => (
   </MUITableRow>
 )
 
+const BurnIcon = ({ color }: { color?: string }) => {
+  if (!color) {
+    color = '#CD5B5B'
+  }
+  return (
+    <LocalFireDepartmentIcon sx={{ color: {color} }} />
```
Comment (Shramp) [inline frontend/src/components/WrapTableRow.tsx:57]: you can do `<LocalFireDepartmentIcon sx={{ color }} />`

Comment (Shramp) [reply frontend/src/components/WrapTableRow.tsx:57]: tbh I'm surprised this works, becuse you're providing an object to the color property of the sx object, which should be a string, not an obect.

File: frontend/src/components/WrapTableRow.tsx [hunk 3]
```diff
@@ -223,17 +243,12 @@ export default function WrapTableRow({ rowItem }: { rowItem: TTableData }) {
       </>
     )
   }
-  // unudited mint
+  // unaudited mint
```
Comment (Shramp) [inline frontend/src/components/WrapTableRow.tsx:246]: good catch!

**Review by Shramp — APPROVED** [review]
Looks great! a few code style suggestions, but nothing blocking. Thank you for doing this!

File: frontend/src/components/CopyableField.tsx (hunk 1 above)
Comment (holtzman) [reply frontend/src/components/CopyableField.tsx:30]: Nice! Thanks for the tip.

**Pull request merged**